airflow/executors/kubernetes_executor.py

```diff
@@ -241,6 +245,69 @@ def _get_security_context_val(self, scontext: str) -> Union[str, int]:
         else:
             return int(val)
 
+    def _parse_extra_volume_mounts(self) -> Dict[str, Dict[str, Any]]:
+        res: Dict[str, Dict[str, Any]] = {}
+
+        extra_volume_mounts = conf.get(self.kubernetes_section, 'extra_volume_mounts')
+
+        if extra_volume_mounts:  # pylint: disable=too-many-nested-blocks
+
+            try:
+                res = json.loads(extra_volume_mounts)
+            except JSONDecodeError as e:
+                raise AirflowConfigException(
+                    'Error parsing config option'
+                    ' `extra_volume_mounts`: {}.'.format(e))
+
+            for pvc_name, pvc_settings in res.items():
+                if 'mount_path' not in pvc_settings:
```

Comment:
Can you please create another helper function to handle the iterations of this for loop? There's a lot of nesting happening here

Reply:
Yeah, no problem.